cmd/quota_show_test.go

```diff
@@ -0,0 +1,272 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"reanahub/reana-client-go/pkg/config"
+	"reanahub/reana-client-go/pkg/displayer"
+	"strings"
+	"testing"
+
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+var quotaShowServerPath = "/api/you"
+
+func TestQuotaShow(t *testing.T) {
+	successResponse := `{
+		"quota": {
+			"cpu": {
+				"health": "healthy",
+				"usage": {
+					"human_readable": "1m 5s",
+					"raw": 10
+				},
+				"limit": {
+					"human_readable": "10m 50s",
+					"raw": 100
+				}
+    		},
```

Comment:
Cosmetics: white space to be cleaned here

![whitespace](https://user-images.githubusercontent.com/517546/185939941-c276e423-2def-4861-93b2-55d69298ae85.png)